assets/src/edit-story/utils/pointerEventsCss.js

```diff
@@ -21,10 +21,7 @@ import { css } from 'styled-components';
 
 const PointerEventsCss = css`
```

Comment:
This actually shouldn't be with a capital `P` - it's just a variable, not a component.

Reply:
I was following `ElementWithPosition` and similar. Should we just redo all of these?

Reply:
Yes, I believe so. The capitalization is confusing, it might cause someone to think it's a valid component, which it definitely isn't.

Reply:
Cool. I renamed `pointerEventsCss` and filed #151 for the rest.